packages/design-tokens/src/tokens/functional/components/river/river.json

```diff
@@ -3,13 +3,13 @@
     "River": {
       "spacing": {
         "inner": {
-          "value": "var(--base-size-24)",
+          "value": "var(--base-size-40)",
```

Comment:
Just double-checking that we intend to update _all_ variations of river, not just breakout? I can see from the snapshots that this affects the non-breakout version also.

Reply:
I just noticed that too, I think it would be better if I create RiverBreakout inner specific token so it doesn't affect the River spacing